src/config/dark-sites.config

```diff
@@ -120,6 +120,7 @@ bugmenot.com
 buildbot.orphis.net
 bungie.net
 byte.co
+curiositystream.com
```

Comment:
You have to correctly arrange the site.

Reply:
sorry it should now be in the right place